src/pages/index.tsx

```diff
@@ -29,39 +27,49 @@ const Home: NextPage = () => {
           <option>Popular</option>
         </select>
       </div>
-      {blogPostDummyData.map(
-        ({ id, name, title, lastUpdated, imageUrl, content, comments }) => (
-          <>
-            <div key={id} className="mb-6">
-              <BlogPost
-                id={id}
-                name={name}
-                title={title}
-                lastUpdated={lastUpdated}
-                imageUrl={imageUrl}
-                content={content}
-                comments={comments.length}
-              />
-            </div>
-            <Modal
-              id={id}
-              comments={comments}
-              poster={name}
-              lastUpdated={lastUpdated}
-              post={content}
-              posterAvatarUrl={imageUrl}
-            />
-          </>
-        )
-      )}
+      {blogPosts.data?.length === 0
+        ? "Nothing to see here"
+        : blogPosts.data?.map(
+            ({
+              id,
+              title,
+              updatedAt,
+              content,
+              comments,
+              user: { name, image },
+            }) => (
+              <>
+                <div key={id} className="mb-6">
+                  <BlogPost
+                    id={id}
+                    name={name as string}
+                    title={title}
+                    lastUpdated={updatedAt.toISOString()}
+                    imageUrl={image as string}
+                    content={content}
+                    comments={comments.length}
+                  />
+                </div>
+                <Modal
+                  id={id}
+                  title={title}
+                  comments={comments}
+                  poster={name as string}
+                  lastUpdated={updatedAt.toISOString()}
+                  post={content}
+                  posterAvatarUrl={image as string}
+                />
+              </>
+            )
+          )}
       {/* will use this later to map to the blog post cards */}
```

Comment:
tag the subtask/story number here so we don't forget

Reply:
thanks for the catch, I removed this comment in this PR since this is what was implemented